.env.example

```
@@ -27,6 +27,8 @@ REDIS_PASSWORD=
 # ── Reconciliation ────────────────────────────────────────────────────────────
 RECONCILIATION_STALE_MINUTES=30
 RECONCILIATION_CRON=*/15 * * * *
+# ── Data retention / anonymization (#127) ─────────────────────────────────────
+RETENTION_CRON=0 3 * * *
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RETENTION_CRON key should go before the WEBHOOK_SECRET key

(UnorderedKey)